### ISSUE TITLE
About the data normalization and loss

### ISSUE THREAD

**Issue by Xinchengzelin** [wangwenxi-handsome/waymo_sim_agent#1]
1.  how to get the data normalization data? compute mean and std in  all the train data?
2.  Have you  test that if we don't use normalization data, could we use batchnorm or layernorm in the model? which is better?
3.  nll_with_covariances is really hard to understand, could you give some explainations?

**Comment by wangwenxi-handsome** [comment]
1. yep
2. we tested with batchnorm, but normalization is better
3. nll_with_covariances is used in Multipath++ for motion prediction, you
can read the paper to find explanations. In Sim Agents, we only used
min_ade_prob_heading_loss.
